File: 240417/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.js
const fs = require('fs');

const [a, b, c] = fs.readFileSync(0).toString().split(' ').map(Number);

let min = Math.min(a, b);
min = Math.min(min, c);

if(min == a){
    process.stdout.write(1 + " ");
}else{
    process.stdout.write(0 + " ");
}

if(a === b && b === c){
    process.stdout.write('1');
}else{
    process.stdout.write('0');
}

File: 240417/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        int min = Math.min(a, b);
        min = Math.min(min, c);

        if(min == a){
            System.out.print(1 + " ");
        }else{
            System.out.print(0 + " ");
        }
        
        if(a == b && b == c){
            System.out.print(1);
        }else{
            System.out.print(0);
        }


    }
}